manifests/master/sysconfig.pp

```diff
@@ -82,7 +82,7 @@
 
   include 'pupmod::master::service'
 
-  unless (mock == true) {
+  unless $mock {
     if ($server_distribution == 'PE') or defined(Class['puppet_enterprise::profile::master']) {
```

Comment:
unit  test for this fix?

Reply:
I don't have the bandwidth to unit test this right now.

It's a PE testing hack and winds through various tests and, frankly, the fix was trivial.

Reply:
Also, I'm not sure why the existing tests didn't pick it up.